netci.groovy

```diff
@@ -26,10 +26,10 @@ platformList.each { platform ->
 
     // Calculate the build command
     if (os == 'Windows_NT') {
-        buildCommand = ".\\build.cmd -Configuration ${configuration} -Architecture ${architecture} -Targets Default"
+        buildCommand = ".\\build.cmd -Configuration ${configuration} -Targets Default"
```

Comment:
I fixed the groovy script to execute x86 builds on an x86 console, which should be picked up correctly by the scripts and build the x86 versions